.github/workflows/build-test-package.yml

```diff
@@ -167,21 +223,59 @@ jobs:
       max-parallel: 2
       matrix:
         include:
-          - itk-python-git-tag: "v5.1.1.post1"
+          - itk-python-git-tag: "v5.2rc01"
 
     steps:
     - uses: actions/checkout@v2
 
+    - name: 'Specific XCode version'
+      run: |
+        sudo xcode-select -s "/Applications/Xcode_11.7.app"
+
+    - name: Get specific version of CMake, Ninja
+      uses: lukka/get-cmake@v3.18.3
+
     - name: 'Fetch build script'
       run: |
         curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITKPythonPackage/master/scripts/macpython-download-cache-and-build-module-wheels.sh -O
         chmod u+x macpython-download-cache-and-build-module-wheels.sh
+        git clone https://github.com/InsightSoftwareConsortium/ITKPythonPackage.git
+        cd ITKPythonPackage
+        git checkout test
 
     - name: 'Build 🐍 Python 📦 package'
       run: |
         export ITK_PACKAGE_VERSION=${{ matrix.itk-python-git-tag }}
         export MACOSX_DEPLOYMENT_TARGET=10.9
+        git clone https://github.com/InsightSoftwareConsortium/ITKBSplineGradient
+        cp ./macpython-download-cache-and-build-module-wheels.sh ITKBSplineGradient/
+        pushd ITKBSplineGradient
+        git checkout ${{env.ITKBSplineGradient-git-tag}}
+        ./macpython-download-cache-and-build-module-wheels.sh
+        popd
+        cp ITKBSplineGradient/include/* include/
+        git clone https://github.com/InsightSoftwareConsortium/ITKHigherOrderAccurateGradient.git
+        cp ./macpython-download-cache-and-build-module-wheels.sh ./ITKHigherOrderAccurateGradient
+        pushd ITKHigherOrderAccurateGradient
+        git checkout ${{env.ITKHigherOrderAccurateGradient-git-tag}}
+        ./macpython-download-cache-and-build-module-wheels.sh
```

Comment:
Like with the Linux Python package builds we do not want to download the cache multiple times and unpack the cache, which would overwrite previous module build additions.